### ISSUE TITLE
Percent chars cause NOVERB issues in diff mode

### ISSUE THREAD

**Issue by JustinAzoff** [sachaos/viddy#69]
How to reproduce:

```
viddy -d '[ $((RANDOM%2)) -eq 0 ] && echo done || echo 50%'
```

You'll see

```
50%!(NOVERB)
```

in the output.  I guess a string is being passed to something that expects a format string somewhere?

**Comment by sachaos** [comment]
This bug fixed in viddy v1.0.0-rc.1
Thank you for the reproducable example.

https://github.com/sachaos/viddy/issues/117#issuecomment-2291069041

**Comment by sachaos** [comment]
Viddy v1.0.0 is available now.
